deployment/cdk/opensearch-service-migration/package.json

```diff
@@ -1,19 +1,40 @@
 {
-  "name": "opensearch-service-domain-cdk",
+  "name": "@opensearch-project/opensearch-service-migration-cdk",
   "version": "1.0.0",
   "description": "CDK infrastructure for migrating to AWS OpenSearch",
   "license": "Apache-2.0",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "author": {
```

Comment:
Having this `author` section was a requirement from another work stream, is it causing issues to keep?

Reply:
Updated to keep